HorizonEngine/Platform/OpenGL/GLRendererAPI.cpp

```diff
@@ -22,11 +22,17 @@ namespace Hzn
 
 	void GLRendererAPI::submitClear()
 	{
-		glClear(GL_COLOR_BUFFER_BIT);
+		glEnable(GL_DEPTH_TEST);
+		glClear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
 	}
 
 	void GLRendererAPI::drawElements(const std::shared_ptr<VertexArray>& vertexArray)
 	{
 		glDrawElements(GL_TRIANGLES, vertexArray->getElementBuffer()->size(), GL_UNSIGNED_INT, nullptr);
 	}
+
+	void GLRendererAPI::drawTriangles(const std::shared_ptr<VertexArray>& vertexArray)
+	{
+		glDrawArrays(GL_TRIANGLES, 0, 36);
```

Comment:
Why does triangles take in two values, I assumed for triangles it would be 3

Reply:
This function glDrawArrays takes three parameters, first is the primitive mode of drawing, second is the first index of the vertex buffer, and the third element is the size of the vertex buffer.